bliss/encoder.py

```diff
@@ -330,6 +335,23 @@ def _get_loss(self, pred: Dict[str, Distribution], true_tile_cat: TileCatalog):
         return loss_with_components
 
     def _generic_step(self, batch, logging_name, log_metrics=False, plot_images=False):
+        do_data_augmentation = self.data_augmentation.get("do_data_augmentation")
+        data_augmentation_start = self.current_epoch >= self.data_augmentation.get("epoch_start")
+
+        if data_augmentation_start and (logging_name == "train") and do_data_augmentation:
+            deconv_image = None
+            if self.input_transform_params.get("use_deconv_channel"):
```

Comment:
Wouldn't we always want to rotate, shift, etc the deconvolution if one exists and we're rotating the main image?

Reply:
Can the data augmentation be applied automatically to all the channels, i.e., whichever ones are present, without explicitly referencing any one channel? For example, the background channel should be rotated too if the image is rotated. It'd be nice if that were implemented without us having a special case for the background channel. Same from the deconv channel -- it doesn't seem like a special case should be necessary for it.

Reply:
In other words, how about concatenating all the images, background, deconv images, etc along the channel axis beforehand? The operations like `vflip` will operate on all of them automatically with just one call.

Reply:
Sound good